package.json

```diff
@@ -38,8 +38,8 @@
     "test:watch": "npm run test -- --watch"
   },
   "devDependencies": {
-    "@unicode/unicode-15.0.0": "^1.3.1",
-    "emoji-test-regex-pattern": "^2.0.2",
+    "@unicode/unicode-15.0.0": "^1.5.2",
```

Comment:
While we're here, could you please update to `unicode-15.1.0` instead? It was released last month.

Reply:
sure thing, I've already updated the PR